src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp

```diff
@@ -42,10 +42,12 @@ namespace ConstitutiveRelations {
  * E=\frac{9K\mu}{3K+\mu}=\frac{\mu(3\lambda+2\mu)}{\lambda+\mu}
  * \f] and the _Poisson ratio_ \f[
  * \nu=\frac{3K-2\mu}{2(3K+\mu)}=\frac{\lambda}{2(\lambda+\mu)}
- * \f]. Inversely, these relations read: \f[
- * K &=\frac{E}{3(1-2\nu)} \\
- * \lambda &=\frac{E\nu}{(1+\nu)(1-2\nu)} \\
- * \mu &=\frac{E}{2(1+\nu)}
+ * \f]. Inversely, these relations read: 
```

Comment:
can you remove the `.` (or put it at the end of the equation)

Reply:
there should also be blank lines around the equations so clang-format doesn't mess with them in the future.

Reply:
Maybe a quick PR could add rules for equations to our style guidelines? I'm aware of two rules not listed in our style guidelines (https://spectre-code.org/code_review_guide.html): blank lines surrounding equations, prefer align to eqnarray. But maybe there are more?